docs/script.js

```diff
@@ -26,6 +26,6 @@ google.charts.load('current', {packages: ['table', 'controls']});
 $(() => {
   initializeSidebar();
   map = createMap(disasterMetadataPromise);
-  loadNavbarWithPicker(firebaseAuthPromise, 'GiveDirectly damage/poverty map');
+  loadNavbarWithPicker(firebaseAuthPromise, 'Delphi Map');
```

Comment:
Should also change https://github.com/givedirectly/Google-Partnership/blob/master/docs/index.html#L4